docs/kap_proposals/kap_2_import_helm_charts.md

```diff
@@ -0,0 +1,21 @@
+# Import Helm charts
```

Comment:
I'd call this `Helm charts input type` instead. The importing feature per se will be done by dependency management.

Reply:
noted!